### ISSUE TITLE
NTR: Scalp nodules (Skin and mucosa)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8038]
New term request Scalp nodules (hresko_220314153125): Includes localized soft tissue swelling

**Comment by pnrobinson** [comment]
@MickeySegal
can you provide context? Is it this? https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6290287/
Also, in general, the HPO labels like to be in the singular and if the fact that there are many is essential to the phenotypic observation, then the label should be something like Multiple scalp nodules -- please clarify the intended use case

**Comment by MickeySegal** [comment]
https://www.ncbi.nlm.nih.gov/books/NBK482477/
"Patients with white piedra are usually asymptomatic and only seek medical attention because of the grossly visible nodules on the affected hair. Occasionally, the patient complains of pruritus or pain at the involved site. The nodules are usually soft, pasty, and ivory-colored. They are about 0.5 millimeters in size but can coalesce to blanket several millimeters of the hair shaft. Genital and facial hair is more often involved than the scalp and torso hair."

https://www.ncbi.nlm.nih.gov/books/NBK558090/
"Painful, recurrent soft-tissue swellings (flare-ups) that may precede localized heterotopic ossification. This may occur in the form of scalp nodules in infancy, which may be an early or presenting feature."

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000726
- label: Scalp nodule
- parents: [Abnormal scalp morphology (HP:0001965)](https://hpo.jax.org/app/browse/term/HP:0001965)
- pmid: PMID:30607043|PMID:29494084
- def: A small firm lump located in the skin of the scalp that is greater than one cm in diameter.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8038
- synonyms: Scalp nodules(NONE)
